packages/changed-elements-react/src/common/InformationButton.tsx

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
These files in `common` folder are not common nor reusable. Please move everything closer to where these components are used.